plugins/config/lib/configService.js

```diff
@@ -1049,9 +1091,27 @@ function respondWithConfigFile(response, filename, resource, directories, scope,
         else {
           respondWithJsonError(response,"Resource not yet defined",HTTP_STATUS_NO_CONTENT,location);
         }
-      }); 
+      }, overrideJsonProperties); 
     }
     break;
+  case AGGREGATION_POLICY_MERGE:
+    {
+      getOverrideJsonAsync(lastPath,filename,directories,scope,(result)=> {
```

Comment:
This would be wrong - getOverrideJsonAsync doesn't do merge, it does override.

Reply:
i added the call back function to that function based on if it was merge or override